{"name": "Создание заказа с ингредиентами", "status": "broken", "statusDetails": {"message": "KeyError: 'accessToken'", "trace": "new_user_data = {'email': 'michael36@example.net', 'name': 'Daniel Rojas', 'password': '*_6+T1ax$z'}\n\n    @pytest.fixture\n    def new_user(new_user_data):\n        response = requests.post(Urls.REGISTER_URL, json=new_user_data)\n        response_data = response.json()\n    \n>       return new_user_data, response_data['accessToken']\nE       KeyError: 'accessToken'\n\nconftest.py:23: KeyError"}, "start": 1721932937591, "stop": 1721932937591, "uuid": "8f0900f8-7a30-4397-b9fc-01e6bfd66630", "historyId": "4754feb8000fa005d06ba04b83694c7a", "testCaseId": "4754feb8000fa005d06ba04b83694c7a", "fullName": "create_order_test.TestCreateOrder#test_create_order_with_ingredients", "labels": [{"name": "suite", "value": "create_order_test"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "MacBook-Pro-Ola.local"}, {"name": "thread", "value": "40148-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "create_order_test"}]}